fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -223,7 +223,9 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
                 image = AppCompatResources.getDrawable(
                     context,
                     R.drawable.mozac_ic_translate_24,
-                )!!,
+                )!!.apply {
+                    setTint(ContextCompat.getColor(context, R.color.fx_mobile_text_color_primary))
+                },
                 imageSelected =
                 AppCompatResources.getDrawable(
```

Comment:
```suggestion
                imageSelected = AppCompatResources.getDrawable(
```
While we are here, could you also fix this poor formatting from my original PR

Reply:
Done